packages/protocol/contracts/rewards/StakingRewards.sol

```diff
@@ -407,6 +461,29 @@ contract StakingRewards is ERC721PresetMinterPauserAutoIdUpgradeSafe, Reentrancy
     return leverageMultiplier;
   }
 
+  /// @notice Calculate the exchange rate (scaled by 1e18) that will be used to convert the original
+  ///   staked token amount to the `baseStakingToken()` amount
+  /// @param positionType Type of the staked postion
+  function getBaseTokenExchangeRate(StakedPositionType positionType) public view virtual returns (uint256) {
```

Comment:
@landakram mind taking another look at this logic here? Both FIDU and Curve LP tokens have 18 decimals, but I added extra safeguards to make sure the token prices are scaled to the same number of decimal places before calculating the exchange rate.

Reply:
@landakram friendly bump 🙂

Reply:
Looks reasonable to me. Can we add a comment that the return value of `getBaseTokenExchangeRate` is denominated in `MUTIPLIER_DECIMALS`?

Though in retrospect, it's prob fine to always assume 1e18 coming in for both fidu and the curve LP virtual price. They just aren't going to change their decimal places. We can save two SLOADS from `decimals()` by doing this, which seems worthwhile to me. Apologies if I suggested otherwise before.